src/file/FileContext.cc

```diff
@@ -68,7 +69,8 @@ const string SYSTEM_DATA  = "{{SYSTEM_DATA}}";
 		}
 	}
 	// not found in any path
-	throw FileException(filename, " not found in this context");
+	throw FileException("Couldn't find ", filename, " in ", (pathList.size() == 1 ? "" : "any of: "),
+		join(std::views::transform(pathList, [](const auto& p) { return FileOperations::getAbsolutePath(p); }), ", "));
```

Comment:
Maybe we could already cherry-pick this commit on master?

Reply:
Sure! I first wanted to show it to you. Apparently you're happy with it :)